### PR TITLE
move privatesend rpc methods from rpc/masternode.cpp to new rpc/privatesend.cpp

### PR DESCRIPTION
**Pull request by PastaPastaPasta**
Doesn't really make sense imo for those rpc commands to be in masternode.cpp.

Should be just a move only and then adjusting imports

### CODE REVIEW

**Comment by PastaPastaPasta** [comment]
(Hopefully) resolved the compilation errors when wallet is disabled.

File: src/rpc/privatesend.cpp [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+                "  \"max_amount\": xxx,                 (numeric) How many " + CURRENCY_UNIT + " to keep mixed\n"
+                "  \"max_denoms\": xxx,                 (numeric) How many inputs of each denominated amount to create\n"
+                "  \"queue_size\": xxx,                 (numeric) How many queues there are currently on the network\n"
+                "  \"sessions\":                        (array of json objects)\n"
+                "    [\n"
+                "      {\n"
+                "      \"protxhash\": \"...\",            (string) The ProTxHash of the masternode\n"
+                "      \"outpoint\": \"txid-index\",      (string) The outpoint of the masternode\n"
+                "      \"service\": \"host:port\",        (string) The IP address and port of the masternode\n"
+                "      \"denomination\": xxx,           (numeric) The denomination of the mixing session in " + CURRENCY_UNIT + "\n"
```
Comment (codablock) [inline src/rpc/privatesend.cpp]: Large parts of the help text are missing here.

**Comment by charlesrocket** [comment]
Shouldn't this be merged in `15` instead?

**Comment by UdjinM6** [comment]
Hmm.. yeah, probably a good idea. Should remove conflicts for #3265 backport.

**Comment by PastaPastaPasta** [comment]
@UdjinM6 @charlesrocket

Why should this be in 15? It doesn't affect anything, just a refactoring really

**Comment by UdjinM6** [comment]
@PastaPastaPasta just like with bitcoin backports it's better to keep them in order if this allows you to avoid merge conflicts and it's a trivial one so why not?

**Pull request merged**